# Literature Review Worksheet

#### Citation

Brand, AH and Perrimon, N. 1993. Targeted gene expression as a means of altering cell fates and generating dominant phenotypes. *Development* 118 (2):401-415. http://dev.biologists.org/content/118/2/401.abstract

#### Question or issue being addressed

This article is a proof-of-concept demonstration for an experimental method. However, the issue relates to the modularity of gene regulatory elements, the idea that the regulatory sequences of a gene can be removed and artificially placed near a new protein-coding sequence, which they will now regulate in the same time, place and intensity.

#### Main finding

This article does two things. First it presents a new method for expressing a protein using a set of artificial, genetic sequences that have been inserted into the genome by use a transposon vector. Two of these so-called “transgenes” are used. The first transgene uses an enhancer sequence to drive expression of the yeast transcription factor protein, GAL4. The second transgene is a *Drosophila* coding sequence expressed under the control of the yeast enhancer sequence called *UAS*. Alone, neither of these transgenes has any effect on development of the fly. But when separate lines are crossed, so that offspring inherit both transgenes, the enhancer of the GAL4 line drives expression of the protein encoded by the UAS line. This method allows targeted misexpression of proteins. Then, as a further demonstration, the authors misexpress the embryonic transcription factor *even-skipped* in a broader domain than normal. They find that this misexpression represses the signaling molecule encoded by *wingless* and alters segmentation of the fly embryo, which was predicted based on the normal interaction of those genes in the embryo. They also show that the GAL4/UAS method can drive protein misexpression in the adult eye and wing. They conclude that this method could have broad usefulness for developmental and neurobiological studies.

#### Scientific quality

This study shows that modular gene regulatory sequences (“enhancers”) can be copied and used to create artificial transgenes capable of directly targeted misexpression of other proteins. This method is very promising because it will allow researchers to mix and match expression patterns and protein sequences at will. If this method can be adapted for use in other animals (even plants?) it could also be a powerful tool for research in those organisms.

#### Writing quality

As a piece of writing, this article leaves something to be desired. It is very dense with terminology, and definitely not written for the non-specialist. You have to check 4-5 references on previous transgenics work to find definitions of terms like pGawB and plwB. They don’t do a great job of explaining the importance of their proof of principle experiments. However, it does follow a logical organization, even if it isn’t obviously signaled to the reader. The evidence that is presented proceeds from general to more specific demonstrations of the Gal4/UAS system. 

#### Three specific questions I have about this article:

1. They don’t really explain how an “enhancer trap” is made. How do they identify enhancers that drive expression is specific tissues or organs?
2. Why did they choose to look at misexpression of Dras2 in the wing and eye? What’s significant about that protein, when they could have chosen any other?
3. Could this method be used in any other animal? Or is it limited to *Drosophila*?
